e2e-tests/tfm/cypress/e2e/journeys/login/login-with-role.spec.js

```diff
@@ -1,30 +1,25 @@
-import pages from '../../pages';
 import USERS from '../../../fixtures/users';
 import TEAMS from '../../../fixtures/teams';
 import relativeURL from '../../relativeURL';
 
 context('Login to tfm with specific roles', () => {
   const findOneUserByTeamId = (teamId) => USERS.find((user) => user?.teams?.includes(teamId));
 
-  beforeEach(() => {
-    pages.landingPage.visit();
-  });
-
   const nonPdcTeams = Object.values(TEAMS).filter((team) => !team.includes('PDC'));
   nonPdcTeams.forEach((team) => {
-    it(`should redirect to /deals after a login for users in '${team.id}' team`, () => {
+    it(`should redirect to /deals after a login for users in '${team}' team`, () => {
       const userInTeam = findOneUserByTeamId(team);
-      cy.login(userInTeam);
+      cy.mockLogin(userInTeam);
 
       cy.url().should('eq', relativeURL('/deals'));
     });
   });
 
   const pdcTeams = Object.values(TEAMS).filter((team) => team.includes('PDC'));
   pdcTeams.forEach((team) => {
-    it(`should redirect to /utilisation-reports after a login for users in '${team.id}' team`, () => {
+    it(`should redirect to /utilisation-reports after a login for users in '${team}' team`, () => {
```

Comment:
`team team` 😄  maybe can rename (i'm not sure what the `team` variable displays)

Reply:
We call it `team` everywhere, including tfm-users collection `user` object. User has attribute `teams` (aka groups, aka roles)

Reply:
I've updated the `team` variable to be `teamName`, think it's clearer